pyprep/reference.py

```diff
@@ -260,7 +230,7 @@ def robust_reference(self):
         # Remove reference from signal, iteratively interpolating bad channels
         raw_tmp = raw.copy()
         iterations = 0
-        noisy_channels_old = []
+        previous_bads = set()
         max_iteration_num = 4
```

Comment:
perhaps adding a comment here that this number is derived from the paper? Otherwise it may appear magical

Reply:
It's actually a configurable parameter in MatPREP (4 is just the default): should we make it a parameter in PyPREP as well?

Reply:
Yes, I think that'd be best 👍